src/Makefile.base

```diff
@@ -61,8 +61,10 @@ endif
 ifneq ($(BOARD), frdm_k64f)
 	obj-$(ZJS_AIO) += zjs_aio.o
 	obj-$(ZJS_GROVE_LCD) += zjs_grove_lcd.o
-	obj-$(ZJS_I2C) += zjs_i2c.o
+	obj-$(ZJS_I2C) += zjs_i2c_ipm.o
 	obj-$(ZJS_SENSOR) += zjs_sensor.o
+else
+	obj-$(ZJS_I2C) += zjs_i2c.o
 endif
```

Comment:
Instead, we should make i2c_ipm conditional on A101 board, i2c for everything else. A101 is the outlier case here.